vcpkg/maintainers/functions/vcpkg_make_configure.md

```diff
@@ -0,0 +1,162 @@
+    [LANGUAGES <language>...]
+)
+```
+
+To use this function, you must depend on the helper port `vcpkg-make`:
+
+```json
+"dependencies": [
+  {
+    "name": "vcpkg-make",
+    "host": true
+  }
+]
+```
+
+## Parameters
+
+### AUTOCONFIG
+
+Runs autoreconf when set.
+
+### COPY_SOURCE
+
+If specified, the source directory will be copied to the build directory before the configuration process begins. Useful for projects that modify the source tree during configuration or build.
+
+### DISABLE_MSVC_WRAPPERS
+
+By default, `vcpkg_make_configure` supplies wrappers that translate and reorder 'Unix / GCC-style' flags into 'MSVC-style' flags. This accommodates build systems that assume Unix or GCC conventions.
+
+Setting `DISABLE_MSVC_WRAPPERS` disables translation of flags, allowing tools like `cl` and `windres` to receive command-line flags without modifications.
+
+### DISABLE_MSVC_FLAG_ESCAPING
+
+By default, escape characters (e.g., -Xcompiler, -Xlinker) are added before compiler and linker flags when using MSVC. These escape characters are intended for use by the wrappers and libtool to protect flags that might contain spaces, quotes, or other special characters from being misinterpreted. Setting `DISABLE_MSVC_FLAG_ESCAPING` disables this behavior. 
+
+This can be useful in situations where you're using custom build scripts or tools that require specific flag formatting.
+
+>[!NOTE]
+>If you disable the MSVC wrappers using `DISABLE_MSVC_WRAPPERS`, this option (`DISABLE_MSVC_FLAG_ESCAPING`) has no effect because the escape characters are not added in the first place.
+
+### DSIABLE_CPPFLAGS
+
+Disables the extraction of C preprocessor flags (`-D` and `-isystem`) from `CFLAGS` and `CXXFLAGS` into `CPPFLAGS`.
+
+By default, `vcpkg_make_configure` extracts preprocessor relevant flags from the `CFLAGS` and `CXXFLAGS` environment variables into `CPPFLAGS`, because preprocessor flags in vcpkg are normally supplied through compiler settings variables like `VCPKG_C_FLAGS` or `VCPKG_CXX_FLAGS`.
+
+Setting `DISABLE_CPPFLAGS` disables that behavior, leaving preprocessor relevant flags in `CFLAGS` and `CXXFLAGS`.
+
+This can be useful for projects that do not use standard environment variables or have specific preprocessor requirements.
+
+### DISABLE_DEFAULT_OPTIONS
+
+Setting this flag disables the following default configure options provided by `vcpkg_make_configure`. Use this flag if you need full control over the configure command line.
+
+General Defaults:
+* --disable-silent-rules - Makes the build process output more verbose.
+* --verbose - Enables verbose output from the configure script.
+
+Library Linkage:
+* For dynamic library builds: --enable-shared and --disable-static
+* For static library builds: --disable-shared and --enable-static
+
+Installation Directories:
+* Adjusts standard directories (bin, lib, share, etc.) to conform with vcpkg's layout, ensuring correct installation paths.
+
+Platform-Specific Tweaks:
```

Comment:
This section seems like it should be summarized as "Additional mysterious platform tweaks." -- the text below just says "On windows we apply windows tweaks. On non-windows we apply non-windows tweaks".

It would be much better to give some indication of what is actually done. Are all arguments scanned for absolute paths? Are backslashes converted to forward slashes?